tests/canary/scripts/push_stats_to_cloudwatch.py

```diff
@@ -0,0 +1,93 @@
+import boto3
+from datetime import datetime
+import xml.etree.ElementTree as ET
+import os
+
+
+xml_path = "../canary/integration_tests.xml"
+
+
+def readXML_and_publish_metrics_to_cw():
+    if os.path.isfile(xml_path):
+        tree = ET.parse(xml_path)
+        testsuite = tree.find("testsuite")
+        failures = testsuite.attrib["failures"]
+        tests = testsuite.attrib["tests"]
+        successes = tests - failures
+    else:
+        failures = 0
+        successes = 0
+        tests = 1
+
+    timestamp = datetime.now().strftime("%Y-%m-%dT%H:%M:%S")
+
+    print(f"Failures: {failures}")
+    print(f"Total tests: {tests}")
+
+    # push to cloudwatch
+    cw_client = boto3.client("cloudwatch")
+    codebuild_client = boto3.client("codebuild")
+
+    codebuild_projects = codebuild_client.list_projects()["projects"]
```

Comment:
suggest passing this as an environment variable for the codebuild job and consume it from there instead. See how we get CLUSTER_NAME or CLUSTER_REGION